core/src/test/java/org/elasticsearch/search/suggest/AbstractSuggestionBuilderTestCase.java

```diff
@@ -79,7 +79,7 @@
 
 public abstract class AbstractSuggestionBuilderTestCase<SB extends SuggestionBuilder<SB>> extends ESTestCase {
 
-    private static final int NUMBER_OF_TESTBUILDERS = 20;
+    private static final int NUMBER_OF_TESTBUILDERS = 2000;
```

Comment:
Sorry, I think I might have sneaked this in, I like to run a lot of repetions ;-) Can you change it back?